staging/com.scoreSmart/manifest.tt

```diff
@@ -0,0 +1,27 @@
+class @com.scoreSmart {
+  import loader from @org.thingpedia.v2();
+  import config from @org.thingpedia.config.none();
+
+  monitorable list query standings(in req league_id: String #_[prompt="what league standings do you want out check out?"],
+                        out Team: String,
```

Comment:
You should probably declare entity types for teams and leagues, and return that here, so you can return both the team ID and the team name (resp. league ID and league name).
Look at Spotify or Yelp for examples of entity type declarations.